eox_nelp/tests/test_principal_views.py

```diff
@@ -0,0 +1,27 @@
+"""
+Test views file.
+"""
+
+from django.test import TestCase
+from django.urls import reverse
+
+import eox_nelp
+
+
+class EOXInfoTestCase(TestCase):
+    """
+    Test for eox-info view.
+    """
+
+    def test_view_info_accesible(self):
+        """
+        Should get a successful answer
+        """
```

Comment:
I like to add expected behavior section
```
        Expected behavior:
            - Return expected content.
            - Status code 200.
        """
```

Reply:
https://github.com/eduNEXT/eox-nelp/pull/26/commits/3defcd0ab909bac5fe867e046b75a86a4472bf7c